src/ol_infrastructure/infrastructure/mongodb_atlas/Pulumi.infrastructure.mongodb_atlas.mitx-staging.Production.yaml

```diff
@@ -0,0 +1,15 @@
+---
+secretsprovider: awskms://alias/infrastructure-secrets-production
+encryptedkey: AQICAHgQalNS7T35ZlcFdhF0QuKeiJAbXMUbm01pjGwHEsjRCgEH/Ldr/W/9AqE0WvSsooQQAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMQO1uRm9iFyL7OgvmAgEQgDtQSaxLII38vLduNnCVTKk4KSMYRusv1pvOrboW/ToGQWaNWjR4V81HqDkKWsN0tkrl8pVvVVTOc3E+8A==
+config:
+  aws:region: us-east-1
+  consul:address: https://consul-mitx-staging-production.odl.mit.edu
+  consul:scheme: https
+  environment:business_unit: residential-staging
+  environment:target_vpc: residential_mitx_staging_vpc
+  mongodb_atlas:disk_autoscale_max_gb: '200'
+  mongodb_atlas:disk_size_gb: 100
```

Comment:
No quotes?

Reply:
Not required. It's the yaml formatter that keeps adding that in.